### DISCUSSION TITLE
[1367550]Kingdom Rush Vengeance {王国保卫战4}

### DISCUSSION THREAD

**Discussion by XiaoAQWQ** [pjy612/SteamManifestCache#608] [Share]
### Steam 游戏ID

1367550

### 密钥列表

```hash
1367550;019f77577cd268395f24eacdb6b0c6f49ad9c612db96ab634e8ffaf73e742f2b
1367552;6dd002ff1dbc19423c66d76b1bcf01fa35234ab22f5e041322d8ede40b5de471
```

### 清单附件

[清单.zip](https://github.com/pjy612/SteamManifestCache/files/15032809/default.zip)

**Comment by XiaoAQWQ** [comment]
新人第一次分享，不知道对不对，购买后看到文件有1367550就一起复制上了

**Reply by pjy612** [reply]
一般这种和 appid 一样的 key 是给 创意工坊的清单解密用的。
有用 但不大 能弄到创意工坊的清单的话就有用，对于某些人而言
